### ISSUE TITLE
Problem about overload of "train()" function of a subclass in nn.Module

### ISSUE THREAD

**Issue by fujiawei0724** [KindXiaoming/pykan#108]
Thanks for your great work. It paves another way different from MLP. However, I find that the authors try to overload the "train()" function of the "KAN" class, which is a subclass of nn.Module. This overload will cause the "train" and "eval" mode transformation of the total failed. I want to figure out whether the author give another way to change the train of eval mode of the mode? Thanks for your reply.

**Comment by STQ-AmadeusUser** [comment]
The same question.

**Comment by Papillon-forest** [comment]
The same question.
